src/ui/MediumVideoPreview/index.tsx

```diff
@@ -35,7 +35,7 @@ const MediumVideoPreview = ({
     </div>
   ) : (
     <div className="movie-preview">
-      <div>
+      <div className="movie-image">
         <img src={shortsImage} alt={shortsTitle} />
```

Comment:
Prop names should be updated from `shorts` to `mdVideo` and then in details you can use specific prop name like `shorts` or `movie`